### PR TITLE
basic keyboard layout

### PR DESCRIPTION
**Pull request by lucianoeli**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,19 +1,216 @@
...
-        tools:context=".MainActivity">
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:id="@+id/activity_main"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:gravity="center"
+              android:orientation="vertical">
 
     <TextView
+            android:id="@+id/resultLabel"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: use android names convention..  words in lower case and separated by "_"

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,19 +1,216 @@
...
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="Hello World!"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintLeft_toLeftOf="parent"
-            app:layout_constraintRight_toRightOf="parent"
-            app:layout_constraintTop_toTopOf="parent"/>
+            android:text="@string/txt_main_activity_result_label_value"
+            android:textSize="20sp"/>
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: use dimens pls

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,19 +1,216 @@
...
+                android:textSize="@dimen/main_activity_number"
+
+        />
+        <Button
+                android:id="@+id/resetButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/txt_main_activity_reset_btn_label"
+                android:textSize="@dimen/main_activity_number"
+
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: remove unnecessary space

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -1,19 +1,216 @@
...
+                android:layout_toRightOf="@id/deleteButton"
+        />
+        <Button
+                android:id="@+id/oneButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="1"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: use strings pls

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (lucianoeli) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (lucianoeli) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (lucianoeli) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (lucianoeli) [reply app/src/main/res/layout/activity_main.xml]: done

**Review by SartiMau — APPROVED** [review]

**Review by capraber — APPROVED** [review]

**Pull request merged**